homeassistant/components/qvr_pro/camera.py

```diff
@@ -0,0 +1,70 @@
+"""Support for QVR Pro streams."""
+
+from homeassistant.components.camera import SUPPORT_STREAM, Camera
+
+from .const import DOMAIN, SHORT_NAME
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the QVR Pro camera platform."""
+    client = hass.data[DOMAIN]["client"]
+
+    entities = []
+
+    for channel in hass.data[DOMAIN]["channels"]:
+        entities.append(QVRProCamera(**channel, client=client))
+
+    add_entities(entities)
+
+
+class QVRProCamera(Camera):
+    """Representation of a QVR Pro camera."""
+
+    def __init__(self, name, model, brand, channel_index, guid, client):
+        """Init QVR Pro camera."""
+
+        self.camera_name = f"{SHORT_NAME} {name}"
+        self.camera_model = model
+        self.camera_brand = brand
```

Comment:
Doing so results in an error since the parent class defines the property via a decorated method.  Is there some other way to set the property other than overriding the decorated method?

![image](https://user-images.githubusercontent.com/453840/74889814-cae4d800-5347-11ea-9701-dc13009180ee.png)

Reply:
You could add `_` as a prefix so `self._brand`.